content/docs/04-clusters/06-cluster-management/14-noc-ui.md

```diff
@@ -24,13 +25,13 @@ The private cloud clusters must set the location explicitly. To set the location
 * Login to the Palette console.
 
 
-* Select the cluster to which the location needs to be updated and go to the `cluster details` page of the cluster.
+* Select the cluster to which the location needs to be updated and go to the **cluster details** page of the cluster.
 
 
-* Open `Settings` and then `Cluster Settings`.
+* Open **Settings** and then **Cluster Settings**.
 
 
-* Select `Location` from the left menu and set the location of that cluster to the wizard and save the changes.
+* Select **Location** from the left menu and set the location of that cluster to the wizard and save the changes.
```

Comment:
```suggestion
* Select **Location** from the left menu, set the cluster's location, and save the changes.
```
"To that wizard" sounds awkward because although we are using a wizard, it's not what we save the changes to. The wizard is just a UI tool to help the user.